boards/itsybitsy_m0/Cargo.toml

```diff
@@ -6,43 +6,70 @@ description = "Board Support crate for the Adafruit ItsyBitsy M0"
 keywords = ["no-std", "arm", "cortex-m", "embedded-hal"]
 categories = ["embedded", "hardware-support", "no-std"]
 license = "MIT OR Apache-2.0"
-edition = "2021"
 repository = "https://github.com/atsamd-rs/atsamd"
 readme = "README.md"
+edition = "2021"
+resolver = "2"
```

Comment:
This is probably a nit, but `edition = "2021"` implies `resolver = "2" `, so it's not actually needed here.

Reply:
I was not aware of that thanks. This PR has been lying around for a while now and it is not aligned with the current master anymore, also mostly because I switched from rust/itsibitsy_m0 to C++/ESP32-C3 for my project (blame on me). I am also not sure anymore how well accepted my proposal was and if somebody is interested at all yet in this change. If not, I don't take it personally and I can close this PR (I had forgotten about this actually...). If yes, I can try to realign it again, but then it needs to be merged before the differences get too big again.

Reply:
Thanks! I will see if I find some time in the couple of days to get it done.

Reply:
I removed that `resolver = "2"` line. But it is there because it is in the feather_m0 code as well, from which it is copied ;)

Reply:
Heh thanks, like I said it's really a nit but thanks for cleaning it up nonetheless!